ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Monitor.hs

```diff
@@ -57,6 +58,16 @@ import Ouroboros.Network.PeerSelection.Types
 governor_BOOTSTRAP_PEERS_TIMEOUT :: DiffTime
 governor_BOOTSTRAP_PEERS_TIMEOUT = 15 * 60
 
+-- | Constants establishing target number of big ledger peers
+-- in TooOld (stale chain) state when in Genesis mode
+--
+governor_GENESIS_KNOWN_BIG_LEDGER_PEERS :: Int
+governor_GENESIS_KNOWN_BIG_LEDGER_PEERS = 50
```

Comment:
Use 100 instead of 50. Know peers are dirt cheap and it means that we have more untested peers to choose from when churning.